File: app/src/main/java/com/example/githubrepo/MyApplication.java
package com.example.githubrepo;

import android.app.Application;

import com.example.githubrepo.services.module.AppModule;

import dagger.ObjectGraph;

/**
 * Created by sarah_neo on 20/02/2017.
 */

public class MyApplication extends Application {

    private ObjectGraph objectGraph;

    @Override
    public void onCreate() {
        super.onCreate();
        objectGraph = ObjectGraph.create(new AppModule(this));
        objectGraph.inject(this);
    }

    public ObjectGraph getObjectGraph() {
        return objectGraph;
    }

    public void inject(Object obj) {
        objectGraph.inject(obj);
    }
}

File: app/src/main/java/com/example/githubrepo/MyApplication.kt
package com.example.githubrepo

import android.app.Application
import com.example.githubrepo.services.module.AppModule
import dagger.ObjectGraph

class MyApplication : Application() {

    lateinit var objectGraph: ObjectGraph
        private set

    override fun onCreate() {
        super.onCreate()
        objectGraph = ObjectGraph.create(AppModule(this))
        objectGraph.inject(this)
    }

    fun inject(obj: Any) {
        objectGraph.inject(obj)
    }
}